### DISCUSSION TITLE
DPM++ 2M Karras throws nan check frequently

### DISCUSSION THREAD

**Discussion by eneskuluk** [lshqqytiger/stable-diffusion-webui-directml#68] [Q&A]
It throws when using DPM++ 2M Karras: `A tensor with all NaNs was produced in Unet. Use --disable-nan-check commandline argument to disable this check.` inconsistently. It might produce a few images before throwing, just throwing this error even at the first image.  Disabling check only yields to time loss since produced images become only white or black. 
6700XT, 512x512 image.
command line args= --medvram --no-half --opt-sub-quad-attention --sub-quad-q-chunk-size 512 --sub-quad-kv-chunk-size 512 --sub-quad-chunk-threshold 80

**Comment by Miraihi** [comment]
It happens to me, but not frequently. I've heard that `opt-sub-quad-attention` is the culprit here.

**Comment by diffusersd** [comment]
> It throws when using DPM++ 2M Karras: `A tensor with all NaNs was produced in Unet. Use --disable-nan-check commandline argument to disable this check.` inconsistently. It might produce a few images before throwing, just throwing this error even at the first image. Disabling check only yields to time loss since produced images become only white or black. 6700XT, 512x512 image. command line args= --medvram --no-half --opt-sub-quad-attention --sub-quad-q-chunk-size 512 --sub-quad-kv-chunk-size 512 --sub-quad-chunk-threshold 80

It was throwing to me as well, after applying following it is much more stable, using same gpu: --medvram --no-half-vae --opt-sub-quad-attention --sub-quad-q-chunk-size 512 --sub-quad-kv-chunk-size 512 --sub-quad-chunk-threshold 80 --opt-split-attention-v1
